tests/unit/components/createDialog.ts

```diff
@@ -0,0 +1,113 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import { VNode } from '@dojo/interfaces/vdom';
+import createDialog from '../../../src/components/dialog/createDialog';
+
+registerSuite({
+	name: 'createDialog',
+
+	construction() {
+		const dialog = createDialog({
+			properties: {
+				id: 'foo',
+				modal: true,
+				open: true,
+				title: 'dialog',
+				underlay: true,
+				closeable: true
+			}
+		});
+		assert.strictEqual(dialog.properties.id, 'foo');
+		assert.isTrue(dialog.properties.modal);
```

Comment:
Would be good to get some error `hints` here as there's multiple assertions. Would make it hard to quickly work out which tests are failing and why otherwise.